wdl/tasks/MASSeq.wdl

```diff
@@ -33,7 +33,7 @@ task RemoveMasSeqTruncatedReads {
         boot_disk_gb:       10,
         preemptible_tries:  3,
         max_retries:        1,
-        docker:             "us.gcr.io/broad-dsp-lrma/lr-transcript_utils:0.0.11"
```

Comment:
I noticed you've two versions before this PR 11 and 13.
Can you make sure there's no fragmentation after this PR?

Reply:
Sure.  I'll try to unify them.